src/ucp/wireup/wireup.c

```diff
@@ -889,13 +889,14 @@ static void ucp_wireup_print_config(ucp_context_h context,
 }
 
 int ucp_wireup_is_reachable(ucp_worker_h worker, ucp_rsc_index_t rsc_index,
-                            const ucp_address_entry_t *ae)
+                            const ucp_address_entry_t *ae, int test_iface)
```

Comment:
need to change the function to ucp_wireup_is_reachable(ep, rsc_index, ae) - which will check reachability according to ep flags. cm lane would already exist, if applicable, since we already received the remote address.